packages/common-all/src/fuse.ts

```diff
@@ -132,6 +95,18 @@ export type SerializedFuseIndex = ReturnType<
 
 type FuseEngineOpts = {
   mode?: DEngineMode;
+  /** If specified must be within 0-1 range. */
+  fuzzThreshold: number;
+};
+
+export const getThresholdValue = (configThreshold: number) => {
+  // Setting it to fallback threshold value in case configuration is incorrect.
+  let threshold = 0.2;
+  if (configThreshold >= 0 && configThreshold <= 1) {
+    threshold = configThreshold;
```

Comment:
nitpick: instead of assignment, can we return the default threshold if the threshold is bad? otherwise, return the regular threshold value

Reply:
Sure will change to just return. Although, that can be up for debate on what's better given that its typically suggested go towards limiting return statements from the function to limit the escape points and hence typically making it easier to debug (this function is as simple as it comes so a bit of moot point on debugging).